packages/nx-plugin/src/generators/configuration/__snapshots__/code-pushup-config.integration.test.ts.snap

```diff
@@ -0,0 +1,20 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`generateCodePushupConfig options > should create code-pushup.config.ts with options in tree 1`] = `
+"import type { CoreConfig } from '../dist/packages/models;
+import * as myPlugin from my-plugin";
+import {myPluginCategory} from "my-plugin";
```

Comment:
This doesn't look like valid syntax, there are missing opening and closing quotes in the imports.

I would suggest replacing `.toMatchSnapshot()` with [`.toMatchFileSnapshot(...)`](https://vitest.dev/api/expect.html#tomatchfilesnapshot), that way the output file can have a `.ts` extension and it's easier to see if it's correct.

Reply:
Done